api/cpp/README.md

```diff
@@ -120,6 +120,9 @@ FetchContent_MakeAvailable(Slint)
 add_executable(my_application main.cpp)
 target_link_libraries(my_application PRIVATE Slint::Slint)
 slint_target_sources(my_application my_application_ui.slint)
+if (WIN32)
+    add_custom_command(TARGET my_application POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_RUNTIME_DLLS:my_application> $<TARGET_FILE_DIR:my_application> COMMAND_EXPAND_LISTS)
```

Comment:
maybe this deserve a small command explaining what this does.  I'm afraid this would be black magic for most people otherwise.

Reply:
Ok, done. I'll also add this to the C++ template change then.